ci/build-conda-package.sh

```diff
@@ -22,7 +22,9 @@ echo "building ${pkgname} ${pkgversion}"
 # build conda for default OS and then cross-compile for other OS
 mkdir ~/conda-builds/
 conda package --pkg-name "$pkgname" --pkg-version "$pkgversion"
-conda convert --platform all "./yhat-${pkgversion}-py27_0.tar.bz2" -o ~/conda-builds/
```

Comment:
Would be nice to add a log entry `echo "Converting to windows has been removed and no longer available"` just in case.